python/hsml/engine/model_engine.py

```diff
@@ -157,7 +157,7 @@ def _set_model_version(
         return model_instance
 
     def _build_artifact_path(self, model_instance, artifact):
-        artifact_path = model_instance.path + "/" + artifact
+        artifact_path = "{}/{}".format(model_instance.version_path, artifact)
```

Comment:
Is the artifact path built correctly? Just to make sure that we aren't missing the extra _/Artifacts/_ level.
Artifacts are stored in the `Artifacts` directory in the model version directory. For example, `/Projects/<p>/Models/<model_name>/<model_version>/Artifacts/<artifact_version>/<zip_file>.zip`.

Reply:
Okay so bad naming, by artifact I mean for example the `environment.yml` file or `input_example.json`, this is not related to those artifacts. I'll call it resource instead.